asterisk.logrotate

```diff
@@ -23,3 +24,15 @@
     	/usr/sbin/asterisk -rx 'logger reload' > /dev/null 2> /dev/null
 	endscript
 }
+/var/spool/mail/asterisk {
```

Comment:
I don't see the reason to rotate a mailbox

Reply:
It was already in logrotate, but it didn't work because of permissions. I don't know if it is necessary